README.md

```diff
@@ -242,4 +242,4 @@ Copyright (c) 2016-2017 New Relic, Inc. All rights reserved.
 [8]:  https://docs.newrelic.com/docs/infrastructure/integrations/nginx-integration-new-relic-infrastructure
```

Comment:
Could you please update the README with correct instructions for Windows? Right now it says it's not supported.